pudl_scrapers/spiders/eipinfrastructure.py

```diff
@@ -19,12 +20,21 @@ def start_requests(self):
         output_root = settings_output_dir / "eipinfrastructure"
         self.output_dir = new_output_dir(output_root)
 
-        yield Request("https://environmentalintegrity.org/download/eip-emissions-increase-database/?wpdmdl=13864&refresh=619e7743265e61637775171")
+        yield Request("https://environmentalintegrity.org/download/eip-emissions-increase-database/")
 
     def parse(self, response):
         """Parse the downloaded EIP Infrastructure excel file."""
+        download_url = response.xpath("//a[@class='wpdm-download-link download-on-click btn btn-primary ']").xpath("@data-downloadurl").get()
```

Comment:
I don't fully understand how all the xpath stuff works, but this looks like The Button! Is it getting a valid Excel file back? I saw some test failures.

Reply:
Yeah, the failures are in the zenodo repo. The scraping might be brittle but I'm not sure of an alternative :/